sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -482,56 +490,65 @@ static size_t max_available(void) {
 }
 
 /**
- * Find the next certificate object in the receive perso buffer and copy it out
- * to the passed in location.
+ * Find the next certificate perso LTV object in the receive perso buffer and
```

Comment:
Wondering what's LTV? Since all the object types are in tlv (tag-length-value) AFAIK.

Reply:
Length-Tag-Value vs Tag-Length-Value. In way it is both the same here. We should align on the terminology here but it is referring to the same thing in this case.